cpp/src/Slice/Grammar.y

```diff
@@ -2216,22 +2216,22 @@ type
 {
     if (unit->compatMode())
     {
-        $$ = unit->optionalBuiltin(Builtin::KindValue);
+        $$ = unit->optionalBuiltin(Builtin::KindAnyClass);
     }
     else
     {
         $$ = unit->builtin(Builtin::KindObject);
     }
 }
-| ICE_VALUE
+| ICE_ANYCLASS
 {
     if (unit->compatMode())
     {
-        $$ = unit->optionalBuiltin(Builtin::KindValue);
+        $$ = unit->optionalBuiltin(Builtin::KindAnyClass);
```

Comment:
It would be preferable to apply this compat-mode convertion only for `Value` and not `AnyClass`, since `AnyClass` is a not a 3.7 keyword.